slides/how_to_redbrick.html

```diff
@@ -0,0 +1,204 @@
+                <script type='text/template'>
+                ![login screen](./images/MainHome.png)
+                </script>
+                </section>
+        <section data-markdown>
+        	<script type='text/template'>
+                ## Multiplexer
+                Tmux
+		</script>
+		</section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Tmux
+                - Lighter than screen (which <strike>is shit</strike> we don't use)
+                - Better plugins
+                - Scriptable
+                - To set it up
+                 - **~edu/setupTmux**
+                 - rebinds keys ctrl+b > ctrl+a    
+                 - makes it pretty 
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Auto Attach
+                - nano ~/.zlogin
+                        tmux attach -d
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ## IRC Clients
+                irssi or weechat 
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Weechat
+                - Alternative irc client
+                - Some advantages over irrsi
+                - Looks nicer
+                - Easier to install scripts
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ##### Need to compile it from source, but there's a script for that
+                        ~$ ~/edu/compileweechat
+                </script>
+                </section>        
+        <section data-markdown>
+                <script type='text/template'>
+                ##### We need to set our NICK for the server
+                       ~$ nano ~/.weechat/irc.conf
+                 - change “nicks” field to your username under the redbrick section
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                #### Useful Commands
+                - /script
+                 - **iset.pl** - nice interface for changing settings
+                 - **buffers.py** - channel list
+                 - **screen_away.pl** - sets away status when you detach from screen
+                 - **cmdind.pl** -tells you is its command or message
+                 - **grep** - lets you grep in weechat
+                 - **pushover.pl** - notification center
+                - Remember to /save when you update settings
+                - And run /autojoin --run
+
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Weechat Mobile
```

Comment:
this will break if you try to set it up for them tell them to do this in their own time